08_traits_and_reading_bytes.md

```diff
@@ -28,28 +46,39 @@ fn main() {
     let version = u32::from_le_bytes(buffer);
 
     println!("Version: {}", version);
+    println!("Bytes slice: {:?}", bytes_slice);
```

Comment:
Brought this from the playground since this is referenced in the text. Leaving it to the playground alone added friction since I was following the example locally and forced me to open a new webpage.